### PR TITLE
Tidy Thorpe files, add back in line breaks and comments - Maybe overkill

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
There is an error in the thorpe brand on master, some variables have not been properly declared as the CSS lines haven't been properly ended (missing semi)

This fixes those issues and adds back in all the hundreds of lines breaks to make the variables css file more readable.

It also adds back in loads of bootstrap comments (maybe overkill but good practise)

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
None

#### What gif best describes how you feel about this work?
giphy

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [x] :+1:
- [x] UXUI Team :+1:
- [x] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/thorpe-park/src/less/backgrounds.less [hunk 1]
```diff
@@ -4,87 +4,6 @@
...
-//   }
-// }
-
-//
-// Contextual backgrounds
-// --------------------------------------------------
-
 // Inverse background variant mixin
 // Apply this to all background variants that require inverting
 .bg-inverse() {
```
Comment (nickine) [inline brands/thorpe-park/src/less/backgrounds.less]: 🌮

File: brands/thorpe-park/src/less/colours.less [hunk 2]
```diff
@@ -5,18 +5,18 @@
 // --------------------------------------------------
 
 // Core brand colours
-@logo-red:                 rgb(229, 50, 24);
-@logo-orange:              rgb(250,113,22);
-@logo-yellow:              rgb(255, 222, 0);
+@logo-red:            rgb(229, 50, 24);
```
Comment (nickine) [inline brands/thorpe-park/src/less/colours.less]: rgb - space or no space?

File: brands/thorpe-park/src/less/panels.less [hunk 3]
```diff
@@ -3,14 +3,13 @@
...
-    h2,
-    h3 {
+    h2, h3 {
       margin-bottom: @padding-base-vertical;
     }
   }
   &-default {
     border-color: transparent;
-    .product-description{
+    .product-description {
```
Comment (nickine) [inline brands/thorpe-park/src/less/panels.less]: is this app specific??

File: brands/thorpe-park/src/less/colours.less (hunk 2 above)
Comment (jodiedoubleday) [reply brands/thorpe-park/src/less/colours.less]: space - completed

File: brands/thorpe-park/src/less/panels.less (hunk 3 above)
Comment (jodiedoubleday) [reply brands/thorpe-park/src/less/panels.less]: yes... but already there :(

**Pull request merged**